react/features/large-video/components/LargeVideo.web.js

```diff
@@ -72,6 +102,26 @@ class LargeVideo extends Component<Props> {
             </div>
         );
     }
+
+    /**
+     * Creates the custom styles object.
+     *
+     * @private
+     * @returns {Object}
+     */
+    _getCustomSyles() {
+        const styles = {};
+        const { _customBackgroundColor, _customBackgroundImageUrl } = this.props;
+
+        styles.backgroundColor = _customBackgroundColor || DEFAULT_BACKGROUND_COLOR;
```

Comment:
We already have https://github.com/jitsi/jitsi-meet/blob/master/interface_config.js#L6 why not use that? Or remove it, if we don't want to make it customizable that way.

Reply:
Missed that. Thanks!